networkx/readwrite/json_graph/cytoscape.py

```diff
@@ -11,16 +11,42 @@ def cytoscape_data(G, attrs=None):
     Parameters
     ----------
     G : NetworkX Graph
-
+        The graph to convert to cytoscape format
+    attrs : dict or None (default=None)
+        A dictionary containing the keys 'name' and 'ident' which are mapped to
+        the 'name' and 'id' node elements in cyjs format. All other keys are
+        ignored. Default is `None` which results in the default mapping
+        ``{name="name", ident="id"}``.
```

Comment:
Sorry, I was confused.  Here is what it does:
```
>>> _attrs = dict(name="name", ident="id")
>>> attrs = {"name": "My Name", "ident": "My Name", "test": "test"}
>>> attrs.update({k: v for (k, v) in _attrs.items() if k not in attrs})
>>> attrs
{'name': 'My Name', 'ident': 'My Name', 'test': 'test'}
>>> attrs = {"ident": "My Name", "test": "test"}
>>> attrs.update({k: v for (k, v) in _attrs.items() if k not in attrs})
>>> attrs
{'ident': 'My Name', 'test': 'test', 'name': 'name'}
>>> attrs = {"test": "test"}
>>> attrs.update({k: v for (k, v) in _attrs.items() if k not in attrs})
>>> attrs
{'test': 'test', 'name': 'name', 'ident': 'id'}
````

I think the docstring needs to be updated and some examples should be added.  It doesn't seem that attrs is used other than to set ``name`` and ``ident``.  Maybe instead of ``attrs=None`` we should have ``name="name", ident="id"``.  So the signature would be:
```
def cytoscape_data(G, name="name", ident="id"):
```

Hopefully, I am not still confused.

Reply:
No I think you're right - I wasn't sure what the intent was so I was just trying to explain the code, but it's possible I missed what was actually happening as well. As currently constituted, an example with a non-default `attrs` would be good, but I agree that the current interface (passing in a dict with two mandatory keys and ignoring all others) is less clear than just having kwargs.

Reply:
I agree that we should just have two keyword arguments instead of passing in a dict.
I am trying to think of why the author would have chosen this design and it is possible that users often have a dict with many attributes including these two. But even in this case, I think it is minimal work to use:
```python
nx.cytoscape_graph(G, name=attrs["name"], ident=attrs["ident"])
``` 

Changing the input parameters will require a deprecation....